db/schema.rb

```diff
@@ -204,7 +204,7 @@
     t.text "name", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.text "ecf_id"
```

Comment:
can we change the factory to set up `SecureRandom.uuid`?

Reply:
Updated now 👍